src/esp/assets/ResourceManager.cpp

```diff
@@ -1192,13 +1193,16 @@ void ResourceManager::loadMaterials(Importer& importer,
     if (loadedAssetData.assetInfo.requiresLighting &&
```

Comment:
Is requiresLighting, the same as enables lighting? Like can you load an asset that requires lighting into a dark scene? If so don't we need additional checks for the next if statement?